### ISSUE TITLE
UDPPort should not need to be in NodePort port-range when using service type of loadBalancer for jvb

### ISSUE THREAD

**Issue by luclu** [jitsi-contrib/jitsi-helm#22]
When using the loadBalancer setup for jvb, the UDPPort must be changed from the default port 10000 to a port between 30000 and 32767 otherwise this error will appear (referencing the default "service-node-port-range" of the cluster):
```
spec.ports[0].nodePort: Invalid value: 10000: provided port is not in the valid range. \
  The range of valid ports is 30000-32767
```

I suspect that https://github.com/jitsi-contrib/jitsi-helm/blob/main/templates/jvb/service.yaml#L20 should not be defined at all when using service type of loadBalancer for jvb.

**Comment by luclu** [comment]
Quick heads up: Simply do **not** set it.

As port 10000/UDP will default via the template and must not be specified. If specified it will activate the nodePort which will likely collide with kubernete's "service-node-port-range".

**Comment by luclu** [comment]
Unfortunately the template doesn't seem to work the way intended. Will create a basic PR.

**Comment by kpeiruza** [comment]
IMHO it's a bad idea not to define the port range as the nodes should be on a Network Security Group or behind a firewall, so pinning to specific ports makes sense.

I'd suggest using hostPort for JVB instead, thus avoiding port-mapping processes like those used in NodePort.

About LB, it's probably a bad idea to use it for videobridges if you're running more than one in your deployment. How will it handle the user video streams if you aren't connected to an specific JVB?

In our case we use daemonsets to deploy JVBs, so it's easy to pick any "host port":

```
        - containerPort: 9090
          hostPort: 9090
          name: jvb-ws
```

**Comment by spijet** [comment]
Fixed in v1.3.0 release.
